composer.json

```diff
@@ -32,7 +32,7 @@
         "sonata-project/core-bundle": ">=3.13"
     },
     "require-dev": {
-        "jms/serializer-bundle": "^1.0 || ^2.0",
+        "jms/serializer-bundle": "^3.3",
```

Comment:
can we just drop this on a stable branch? there should be a big difference between 2.0 and 3.0

Reply:
It's a dev dependency....